### ISSUE TITLE
Sorting Should Ignore “a,” “an,” and “the” 

### ISSUE THREAD

**Issue by varanasib** [bookwyrm-social/bookwyrm#2678]
**Is your feature request related to a problem? Please describe.**
Unlike the standard manner of sorting book titles, see libraries and calibre, bookwyrm considers initial articles in titles.  So, for example, "The Yiddish Policemen's Union" appears before "Siddhartha."

**Describe the solution you'd like**
Bookwyrm should sort titles in the standard manner, ignoring “a,” “an,” and “the”

**Describe alternatives you've considered**
Struggling to find books in lists sorted by bookwyrm that cannot be placed in the same order as lists from other sources.

**Comment by mouse-reeve** [comment]
I think the approach here is to create a "sort title" field for the title of the book minus any initial articles, and do a best guess at populating it in a migration, and allow users to edit it.

**Comment by Txopi** [comment]
They are instances with many non English contents: Spanish, French, Italian, Portuguese, Finnish, Chinese, Basque...
The English initial articles list should not interfere with other languages or the "sort title" field's value should be set differently depending on the language of the book. For example "A" in Spanish means "To" and "The" can be "El", "La", "Lo", "Los" or "Las".

**Comment by hughrun** [comment]
If the `sort_title` is set at the edition level that should make it easier to set it appropriately to the language of the edition, but the rules about what to ignore/remove will obviously have to be set for each language.

**Comment by mouse-reeve** [comment]
Yes, I think it would definitely be at the edition level. And I was picturing this as a field that can be manually edited, not one that is generated automatically when a book is edited, so we don't have to algorithmically plan around every language variation or unexpected title format.

**Comment by ccamara** [comment]
I believe this has been fixed after #2834. Can this issue be closed?
